azure-pipelines-official.yml

```diff
@@ -328,8 +332,10 @@ stages:
 
     - template: eng/pipelines/insert.yml
       parameters:
-        clientId: $(ClientId)
-        clientSecret: $(ClientSecret)
+        buildUserName: "dn-bot@microsoft.com"
+        buildPassword: $(_DevDivInsertionAccessToken)
```

Comment:
I would quote these, at least to put my paranoia at bay.

Reply:
Added quotes and spun a new build